File: library-java/lib/src/main/java/eu/postyard/templates/BlockType.java
package eu.postyard.templates;

/**
 * The kind of content a template block produces. It determines how values interpolated into the
 * block are escaped during rendering.
 */
public enum BlockType {

  /** Output is HTML; interpolated values are HTML-escaped. */
  HTML,

  /** Output is plain text; interpolated values are emitted verbatim. */
  TEXT
}

File: library-typescript/src/BlockType.ts
/**
 * The kind of content a block produces, which controls how interpolated values
 * are escaped during rendering.
 */
export enum BlockType {
  /** Output is HTML; interpolated values are HTML-escaped. */
  HTML = "html",

  /** Output is plain text; interpolated values are emitted verbatim. */
  TEXT = "text",
}
